runelite-client/src/main/java/net/runelite/client/plugins/StatusBars/StatusBarsOverlay.java

```diff
@@ -0,0 +1,300 @@
+				int offsetPrayerX = (location.getX() - offsetRight.getX());
+				int offsetHealthX = (location.getX() - offsetLeft.getX());
+				int offsetHealthY = (location.getY() - offsetLeft.getY());
+				int offsetPrayerY = (location.getY() - offsetRight.getY());
+				String counterHealthText = Integer.toString(counterHealth);
+				String counterPrayerText = Integer.toString(counterPrayer);
+
+				if (viewport == Viewport.RESIZED_BOTTOM && !widget.isHidden())
+				{
+					offsetPrayerY = (location.getY() - 18 - offsetRight.getY());
+					offsetHealthY = (location.getY() - 18 - offsetRight.getY());
+					height += 20;
+				}
+				else if (viewport == Viewport.RESIZED_BOX && widget.isHidden())
+				{
+					return null;
+				}
+				else
+				{
+					offsetPrayerX = (location.getX() + widget.getWidth() - offsetRight.getX());
+				}
+
+				// Render the HP and Prayer bar
+				renderBar(g, offsetHealthX , offsetHealthY,
+					maxHealth, currentHealth, WIDTH , height,
+					PADDING, healthBar, healthImage.getScaledInstance(imageSize , imageSize , Image.SCALE_AREA_AVERAGING), counter , counterHealthText, counterHealthX);
+				renderBar(g, offsetPrayerX, offsetPrayerY ,
+					maxPrayer, currentPrayer, WIDTH, height,
+					PADDING, prayerBar, prayerImage.getScaledInstance(imageSize , imageSize , Image.SCALE_SMOOTH), counter , counterPrayerText , counterPrayerX);
+
+				// gets the value from ItemStat plugin to build the hp or prayer bar accordingly to the value of the item a player hovers over.
+				WidgetItem[] inventory = queryRunner.runQuery(new InventoryWidgetItemQuery());
+				java.awt.Point mousePos = new java.awt.Point(client.getMouseCanvasPosition().getX(), client.getMouseCanvasPosition().getY());
+				for (WidgetItem item : inventory)
+				{
+					if (item.getCanvasBounds().contains(mousePos))
+					{
+						Effect change = statChanges.get(item.getId());
+						if (change != null)
+						{
+							StringBuilder b = new StringBuilder();
+							StatsChanges statsChanges = change.calculate(client);
+							for (StatChange c : statsChanges.getStatChanges())
+							{
+								if (c.getStat().getName() == "Hitpoints")
+								{
+									foodHealValue = Integer.parseInt(c.getTheoretical());
+								}
+
+								if (c.getStat().getName() == "Prayer")
+								{
+									prayerHealValue = Integer.parseInt(c.getTheoretical());
+								}
+							}
+						}
+					}
+				}
+				// Bar for healing health or payer points
+				renderHealingBar(g, offsetHealthX , offsetHealthY,
+						maxHealth, currentHealth, WIDTH , height,
+						PADDING, foodHealValue, HEAL_COLOR);
+				renderHealingBar(g, offsetPrayerX , offsetPrayerY,
+						maxPrayer, currentPrayer, WIDTH, height,
+						PADDING, prayerHealValue, PRAYER_HEAL_COLOR);
+			}
+		}
+		return null;
+	}
+
+	private void renderBar(Graphics2D graphics, int x, int y, int max, int current, int width, int height, int padding, Color filled, Image image, Color counter_color, String counter, int counter_x)
```

Comment:
format the arguments a bit

Reply:
make the names more clear to what a variable is? or format in which way?